dos/src/main.rs

```diff
@@ -464,26 +582,59 @@ fn main() {
     }
 
     info!("done found {} nodes", nodes.len());
-    let payer = cmd_params
-        .transaction_params
-        .payer_filename
-        .as_ref()
-        .map(|keypair_file_name| {
-            read_keypair_file(&keypair_file_name)
-                .unwrap_or_else(|_| panic!("bad keypair {:?}", keypair_file_name))
+
+    // create client which is used for airdrop and get blockhash
```

Comment:
Outdated comment? I don't think we use any airdrops in this tool.

Reply:
Incidentally, all of the paths that use this client for requests (getLatestBlockhash, etc) will probably fail against a live cluster, like testnet. This is because the discover_cluster/get_multi_client approach is set up for a small test cluster -- it tries to load all the nodes in the cluster into the client list, and also doesn't distinguish those that have open Rpc ports.
That's why I added the TpuClient support to bench-tps. But seems like you're already looking at ways to use BenchTpsClient better, so perhaps this is already on your radar.